js-basics-and-variables/js-basics-and-variables_track-your-savings-goals/index.js

```diff
@@ -14,3 +14,11 @@ Let's build a small program to track your savings goals!
 
 4. Run your JavaScript program to see how much money you'll have saved after the specified time period.
 */
+
+let startingBalance = 3000;
+let monthlyContribution = 1000;
+let months = 12;
+
+let totalSavings = startingBalance + monthlyContribution * months;
```

Comment:
These can be all constant variables as you are not assigning them a new value. We should always use `const` as default.